test/gas-analytics/SpokePool.Relay.ts

```diff
@@ -0,0 +1,71 @@
+import { toBNWei, SignerWithAddress, Contract, ethers, seedWallet } from "../utils";
+import { spokePoolFixture, getRelayHash, getFillRelayParams } from "../fixtures/SpokePool.Fixture";
+
+require("dotenv").config();
+
+let spokePool: Contract, weth: Contract, erc20: Contract;
+let depositor: SignerWithAddress;
+
+// Constants caller can tune to modify gas tests.
+const RELAY_COUNT = 10;
+const RELAY_AMOUNT = toBNWei("10");
+
+function constructRelayParams(relayTokenAddress: string, depositId: number) {
+  const { relayData } = getRelayHash(depositor.address, depositor.address, depositId, 1, relayTokenAddress);
+  return getFillRelayParams(relayData, RELAY_AMOUNT);
+}
+describe("Gas Analytics: SpokePool Relays", function () {
+  before(async function () {
+    if (!process.env.GAS_TEST_ENABLED) this.skip();
+  });
+
+  beforeEach(async function () {
```

Comment:
This might not be as impactful in this test, but I think we should probably do something similar to the above and "warm" the contract by running relays for all these ERC20s first. In this case, we might also want to run some deposits too to ensure the contract doesn't have a 0 balance of all these ERC20s (since I would expect that that would not be the common case in prod).

Reply:
Will do